site/src/sections/author.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+
+// assets import
+
+const Background = "../assets/images/author/Background.png";
+import LeePhoto from "../assets/images/author/Lee.svg";
+import NicPhoto from "../assets/images/author/Nic.svg";
```

Comment:
We already have Lee and Nic's image under assets. Please re-use them, and delete the newly added images.

Reply:
Yes, I tried that but see this affects the quality of the picture. 
![image](https://user-images.githubusercontent.com/84725791/191316517-27101a90-2da0-4c4d-9be5-5767cf216482.png)
That is why I used this in SVG!!

Reply:
If we are not using `StaticImage` component for rendering the image, we should be good to use it.

Reply:
SVGs is a vector-based image format - not to be used for photos, which are best encoded in a rasterized format, specifically, JPG, has the best quality to compression ratios for real-life photos (of people, places, and things).